src/frontend/screens/Library/components/AlphabetFilter/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo, useContext } from 'react'
+import './index.css'
+import LibraryContext from '../../LibraryContext'
+
+const CHARS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ#'.split('')
+
+const AlphabetFilter: React.FC = () => {
+  const {
+    alphabetFilterLetter: currentFilter,
+    setAlphabetFilterLetter: onFilterChange,
+    gamesForAlphabetFilter: allGames
+  } = useContext(LibraryContext)
+
+  const availableChars = useMemo(() => {
+    const chars = new Set<string>()
+    if (!allGames) return chars
+    allGames.forEach((game) => {
+      if (game?.title) {
```

Comment:
not sure I understand this `game?` part, can a `game` variable be undefined? `allGames` is an array of GameInfo objects

also can `game.title` ever be something other than a string? I don't think so, I don't think we need this if at all